src/enforcer.js

```diff
@@ -572,13 +669,18 @@ export class TrustedTypesEnforcer {
    * @param {function(!Function, ...*)} functionBody The wrapper function.
    */
   wrapFunction_(object, name, functionBody) {
+    if (!object) return;
     const descriptor = getOwnPropertyDescriptor(object, name);
     const originalFn = /** @type function(*):* */ (
         descriptor ? descriptor.value : null);
 
     if (!(originalFn instanceof Function)) {
-      throw new TypeError(
```

Comment:
Huh? Does it throw on node? When? isLoggingEnabled is about the TT errors in your application, not about errors in your environment.

Reply:
True. I don't want to throw here because it will likely break in node (I think even domino breaks here). 

Not sure what to do though? Silent return?

Reply:
Reverted back to throwing an error.